packages/critters-webpack-plugin/test/fixtures/additionalStylesheets/index.js

```diff
@@ -0,0 +1,5 @@
+import './style.css';
+
+document.body.appendChild(document.createTextNode('this counts as SSR'));
+
+import('./chunk.js').then();
```

Comment:
If there's a slightly less awkward way to create another chunk, certainly let me know. This was just to generate another stylesheet